src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegService.kt

```diff
@@ -18,7 +22,7 @@ class StegService(
     @Transactional
     fun utførSteg(behandlingId: Long, behandledeSteg: BehandlingSteg, behandlingStegDto: BehandlingStegDto? = null) {
         val behandling = behandlingRepository.hentAktivBehandling(behandlingId)
-        val behandledeStegTilstand = hentBehandledeSteg(behandling, behandledeSteg)
+        val behandledeStegTilstand = hentStegTilstandForBehandlingSteg(behandling, behandledeSteg)
```

Comment:
Jeg tror forrige metodenavn var bedre, kanskje den kan modifiseres til hentBehandledeStegTilstand. Men hentStegTilstandForBehandlingSteg sier ikke at det henter nåværende steg

Reply:
Er ikke helt enig her 🤔  Synes egentlig `hentStegTilstandForBehandlingSteg` er ganske beskrivende for hva metoden gjør. Den henter stegTilstand for steget man spesifiserer i parameter. Jeg synes ihvertfall at `hentBehandledeSteg` høres ut som at man henter alle steg som har blitt behandlet tidligere. Dersom man alltid henter stegtilstand til steget man står på/har kommet til, burde kanskje metoden hete `hentBehandlingstegTilstand`?

Reply:
Er vanskelig å finne et ord for steget man har kommet til/står på 🤔